lte/gateway/c/core/oai/tasks/nas5g/include/ies/M5GPayloadContainer.h

```diff
@@ -12,13 +12,14 @@
 #pragma once
 #include <sstream>
 #include <cstdint>
+#include <string.h>
 #include <SmfMessage.h>
+#define PAYLOAD_CONTAINER_CONTENTS_MAX_LEN 8192
```

Comment:
are use using namespace std in this file? if so please remove `using namespace std` and explicitly use `std::`

Reply:
if you are not using it at all, simply remove it.

Reply:
the namespace is not using , removed it and updated review

Reply:
@ulaskozat,  I have resolved the above review comment nas5g module. but too many changes
                   can I include it in the same review?